src/usecases/createVisit.js

```diff
@@ -1,10 +1,12 @@
+import { SCHEDULED } from "../../src/helpers/visitStatus";
+
 const createVisit = ({ getDb }) => async (visit) => {
   const db = await getDb();
 
   return await db.one(
     `INSERT INTO scheduled_calls_table
-      (id, patient_name, recipient_email, recipient_number, recipient_name, call_time, call_id, provider, ward_id, call_password)
-      VALUES (default, $1, $2, $3, $4, $5, $6, $7, $8, $9)
+      (id, patient_name, recipient_email, recipient_number, recipient_name, call_time, call_id, provider, ward_id, call_password, status)
```

Comment:
Do we need this change anymore since we're defaulting the status value to `scheduled`?

Reply:
I guess not, although making it explicit is perhaps easier to follow from a maintenance perspective.